packages/itwin/tree-widget/src/components/trees/stateless/models-tree/ModelsTreeDefinition.ts

```diff
@@ -89,7 +85,7 @@ export class ModelsTreeDefinition implements HierarchyDefinition {
         ],
       },
     });
-    this._subjectModelIdsCache = new SubjectModelIdsCache(props.imodelAccess);
+    this._subjectModelIdsCache = props.subjectModelIdsCache ?? new SubjectModelIdsCache(props.imodelAccess);
```

Comment:
As far as I can tell, `props.subjectModelIdsCache` is never `undefined`.

```suggestion
    this._subjectModelIdsCache = props.subjectModelIdsCache;
```